docs/deploying-airbyte/local-deployment.md

```diff
@@ -1,65 +1,39 @@
 # Local Deployment
+This page guides you through setting up your Airbyte on your local machine. 
 
-:::info
-These instructions have been tested on MacOS, Windows 10 and Ubuntu 20.04.
+## Prerequisites
+* [Install Docker](https://docs.docker.com/get-docker/) and make sure you have the latest version of `docker-compose` installed.
+* Additionally, for Windows, install [WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/)  and enable Hyper-V Windows Features".
 
-:::
+> **_Note:_** These instructions have been tested on MacOS, Windows 10, and Ubuntu 20.04.
 
-## Setup & launch Airbyte
+## Deploy on Mac and Linux
+To set up Airbyte on your local machine:
 
-- Install Docker on your workstation \(see [instructions](https://www.docker.com/products/docker-desktop)\). Make sure you're on the latest version of `docker-compose`.
-- After Docker is installed, you can immediately get started locally by running:
+1. Clone the Airbyte repository.
+2. Move into the cloned folder (`airbyte`) and start the server with `docker compose`;
 
 ```bash
 git clone https://github.com/airbytehq/airbyte.git
 cd airbyte
-docker-compose up
+sudo docker compose up
 ```
-
-- In your browser, just visit [http://localhost:8000](http://localhost:8000)
-- You will be asked for a username and password. By default, that's username `airbyte` and password `password`. Once you deploy airbyte to your servers, be sure to [change these](/operator-guides/security):
-
-```yaml
-# Proxy Configuration
-# Set to empty values, e.g. "" to disable basic auth
-BASIC_AUTH_USERNAME=your_new_username_here
-BASIC_AUTH_PASSWORD=your_new_password_here
-```
-
-- Start moving some data!
+3. In your browser, visit http://localhost:8000
+> **_Note:_** You will be asked for a username and password. The default username is `airbyte` and the password is `password`.
```

Comment:
@Amruta-Ranade the syntax **_Note_** is native markdown, why not use docsaurus?

Reply:
Agreed. I'll fix the formatting issues for all deployment docs PRs in a follow-up PR